Kafka SqlAvroTest#setupClass

_master_  (commit f003625ac4fe9ab09ca79e6f143703d0b468b3a8)

Failed on Jet OSS PR Builder http://jenkins.hazelcast.com/job/jet-oss-pr-builder/4280/testReport/junit/com.hazelcast.jet.sql.impl.connector.kafka/SqlAvroTest/com_hazelcast_jet_sql_impl_connector_kafka_SqlAvroTest/

The PR: https://github.com/hazelcast/hazelcast-jet/pull/2888

Stacktrace:
```
java.io.IOException: Failed to bind to /0.0.0.0:39136
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:349)
	at org.eclipse.jetty.server.ServerConnector.open(ServerConnector.java:310)
	at org.eclipse.jetty.server.AbstractNetworkConnector.doStart(AbstractNetworkConnector.java:80)
	at org.eclipse.jetty.server.ServerConnector.doStart(ServerConnector.java:234)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
	at org.eclipse.jetty.server.Server.doStart(Server.java:401)
	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:73)
	at com.hazelcast.jet.sql.impl.connector.kafka.SqlAvroTest.setUpClass(SqlAvroTest.java:94)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at com.hazelcast.test.AbstractHazelcastClassRunner$1.evaluate(AbstractHazelcastClassRunner.java:306)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:288)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:282)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:433)
	at sun.nio.ch.Net.bind(Net.java:425)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:223)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74)
	at org.eclipse.jetty.server.ServerConnector.openAcceptChannel(ServerConnector.java:344)
	... 22 more
```

--- a/hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/SqlAvroTest.java
+++ b/hazelcast-jet-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/SqlAvroTest.java
@@ -41,9 +41,7 @@ import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-import java.io.IOException;
 import java.math.BigDecimal;
-import java.net.ServerSocket;
 import java.time.Duration;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
@@ -86,7 +84,7 @@ public class SqlAvroTest extends SqlTestSupport {
         kafkaTestSupport.createKafkaCluster();
 
         Properties properties = new Properties();
-        properties.put("listeners", "http://0.0.0.0:" + randomPort());
+        properties.put("listeners", "http://0.0.0.0:0");
         properties.put("kafkastore.connection.url", kafkaTestSupport.getZookeeperConnectionString());
         SchemaRegistryConfig config = new SchemaRegistryConfig(properties);
         SchemaRegistryRestApplication schemaRegistryApplication = new SchemaRegistryRestApplication(config);
@@ -476,11 +474,5 @@ public class SqlAvroTest extends SqlTestSupport {
                             .fields()
                             .name(fieldName).type().unionOf().nullType().and().stringType().endUnion().nullDefault()
                             .endRecord();
-    }
-
-    private static int randomPort() throws IOException {
-        try (ServerSocket server = new ServerSocket(0)) {
-            return server.getLocalPort();
-        }
     }
 }

